### ISSUE TITLE
Gmail unread counts stuck

### ISSUE THREAD

**Issue by robjhyndman** [wavebox/waveboxapp#518]
* Which version of Wavebox are you using? 3.7.0
* Which Operating System are you using? Ubuntu 16.04

The unread items are stuck on my two gmail accounts. e.g., one of them shows 4 regardless of what the actual number is. I've tried using advanced unread options, but it makes no difference.

**Comment by nmat** [comment]
Hello @robjhyndman,

Did you look at: https://wavebox.io/kb/google-unread-counts when you did the test to fix the unread account?

Does it make any difference at all with how you set it in gmail and in wavebox?

Also can you check what happens if you only use one account if there is a difference in the unread counters?

**Comment by robjhyndman** [comment]
No, that didn't help. I ended up completely uninstalling wavebox, removing any cached files and local config files, and reinstalling. Then adding one account at a time. That worked. I did discover in the process that the problem was with the exported wavebox file. If I imported the file, the problem re-occurred. But if I re-installed each account manually, it worked ok.

**Comment by nmat** [comment]
hmmm is the exported file from a different version of wavebox?

**Comment by Thomas101** [comment]
Occasionally Google reports some phantom counts, there have been a few instances of this recently. It seems that re-adding your account does work though.

Curious that restoring the configuration brings the problem back though. I wonder if the auth tokens that Google issues, correlate to a specific cluster of machines in their data-center.

Do you still have the faulty configuration? How did you restore it, by copying the files manually or by using the inport/export tools?

**Comment by keypey** [comment]
Same problem, 11 "unread" in 3 different accounts.

wavebox: 3.7
debian stretch

wavebox - Synchronized on second(backup) laptop.
 Where is the "cache"?

**Comment by robjhyndman** [comment]
I restored the faulty config by using input/export tools. The export file was from the same version of wavebox as I'd updated to v3.7.0 before doing the export.

**Comment by keypey** [comment]
Export / Import ... I did not notice this possibility.
Thanks - it helped.
It works great!
